oss-internship-2020/libuv/callbacks/callbacks.cc

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "callbacks.h"
```

Comment:
For internal reasons, please add a line comment like this to all your quoted headers (not SAPI ones):
```c++
#include "callbacks.h"  // NOLINT(build/include)
```

Reply:
Should I do this for `#include "gtest/gtest.h"` as well?